apps/verify/index.js

```diff
@@ -13,7 +13,7 @@ module.exports = {
     '/not-found': {},
     '/verify': {
       fields: ['user-email'],
-      next: '/check-inbox'
+      next: '/ima/evidence-upload'
```

Comment:
Could you write it like this so that we don't forget to change it back once the relevant pages have been added:
` // next: '/check-inbox' TO BE REINSTATED ONCE RELEVANT PAGES ADDED
  next: '/ima/evidence-upload' // TO BE REMOVED ONCE RELEVANT PAGES ADDED `

Reply:
Additional comments added